framework/src/iotbus/iotbus_pwm.c

```diff
@@ -117,7 +116,7 @@ int iotbus_pwm_set_duty_cycle(iotbus_pwm_context_h pwm, uint32_t duty_cycle)
 		return IOTBUS_ERROR_UNKNOWN;
 	}
 
-	info->duty = (duty_cycle * 65536) / 100;
+	info->duty = (duty_cycle << 16) / 100;
```

Comment:
We had better keep the original codes, but change it to definition like IOTBUS_PWM_MAX_RESOLUTION(?). I refered IOTBUS_PWM_MAX_PERIOD.

Reply:
IOTBUS_PWM_MAX_RESOLUTION definition looks appropriate. will update.